src/storage/migration/versions/221cfef47173_added_oncascade_delte_for_foreign_keys.py

```diff
@@ -0,0 +1,85 @@
+"""added oncascade delete for foreign keys
+
+Revision ID: 221cfef47173
+Revises:
+Create Date: 2023-03-23 17:05:22.740165
+
+"""
+# pylint: disable=invalid-name,no-member
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '221cfef47173'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
```

Comment:
Sure

Reply:
the script was automatically generated but I can remove those things if you like